draft-rbw-add-encrypted-dns-forwarders.md

```diff
@@ -446,7 +449,7 @@ devices. The CA in the CPE would be limitied to signing only certificates belong
 that home network (using Sections {{<delegated}} or {{<name-constraints}}).
 This allows the CPE to sign certificates for other devices within the network such as
 printers, IoT devices, NAS devices, laptops, or anything else needing to be a server
-on the local network.
+on the local network.  One standard for such a system is {{?BRSKI=RFC8995}}.
```

Comment:
BRSKI is specific to IoT devices attached to a specific network, please see our draft https://datatracker.ietf.org/doc/html/draft-reddy-add-iot-byod-bootstrap-00#section-5.

Reply:
I don't think BRSKI is deployed/supported.

Reply:
removed citation to BRSKI.